.github/workflows/project-build.yml

```diff
@@ -13,16 +13,21 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        ruby: [2.7.4, 3.0.2, 3.1.0]
-        gemfile: [activesupport_5, activesupport_6]
+        ruby: [2.7, '3.0', '3.1', '3.2']
+        gemfile: [activesupport_5, activesupport_6, activesupport_7]
+        exclude:
+          - ruby: '3.2'
+            gemfile: activesupport_5
```

Comment:
Why is excluded?

Reply:
Rails 5.x isn't supposed to be run with Ruby 3.2.  It may run successfully, or it may not, but it's an unsupported configuration.